internal/application/application.go

```diff
@@ -245,6 +245,27 @@ func deleteStagePVC(ctx context.Context, cluster *kubernetes.Cluster, appRef mod
 		PersistentVolumeClaims(deployments.TektonStagingNamespace).Delete(ctx, appRef.MakePVCName(), metav1.DeleteOptions{})
 }
 
+func PreviousStageId(ctx context.Context, cluster *kubernetes.Cluster, appRef models.AppRef) (string, error) {
```

Comment:
```suggestion
// PreviousStageID return the stage ID of the previous build if one exists. It returns an empty string otherwise.
// This method relies on a workload to get the previous id. There is the case that a stage has happened but there
// is no workload (e.g. by calling the "staging" endpoint but not calling the "deploy" endpoint). Since our client
// doesn't support that, this method doesn't support that either.
func PreviousStageID(ctx context.Context, cluster *kubernetes.Cluster, appRef models.AppRef) (string, error) {
```

Reply:
Notice the capitalization of the method name. Needs the callers to changes as well. My vim plugins complain about Go conventions.